getting-started/FAQ.md

```diff
@@ -184,6 +201,18 @@ CONFIG_DEBUG_INFO_BTF=y
   ```
   as we can see here `bpf` is in active lsms
 
+### Enabling BPF-LSM automatically using updater script deployment
+
+**We can apply the following manifest which automatically detects and installs/enabled BPFLSM/AppArmor whichever is needed in kubernetes worker nodes.**
+
+```
+kubectl apply -f https://raw.githubusercontent.com/kubearmor/KubeArmor/main/deployments/controller/updaterscript.yaml
+```
+
+> **Warning:** After running the above script the nodes will restart.
```

Comment:
Should we also mention to uninstall KubeArmor (with the --force flag) before running this?

Reason: A user had KubeArmor with AppArmor on their system and then they wanted to switch to BPF LSM.
They used the manifest and BPF LSM got enabled however some of their pods stayed stuck because of AppArmor annotations.